manual/src/cmds/intf-c.etex

```diff
@@ -936,36 +935,44 @@ parameters, and use one or more calls to the "CAMLxparam" macros for
 the remaining parameters ("CAMLxparam1" to "CAMLxparam5").
 
 The macros "CAMLreturn", "CAMLreturn0", and "CAMLreturnT" are used to
-replace the C
-keyword "return".  Every occurrence of "return x" must be replaced by
-"CAMLreturn (x)" if "x" has type "value", or "CAMLreturnT (t, x)"
-(where "t" is the type of "x"); every occurrence of "return" without
-argument must be
-replaced by "CAMLreturn0".  If your C function is a procedure (i.e. if
-it returns void), you must insert "CAMLreturn0" at the end (to replace
-C's implicit "return").
+replace the C keyword "return"; any function using a `CAMLparam` macro
```

Comment:
The use of <code>`</code> and <code>'</code> throughout this PR doesn't match the way quotations are used in the rest of the file.